web/views/FcRequestStudyBulkView.vue

```diff
@@ -113,9 +129,8 @@ import { mapActions, mapMutations } from 'vuex';
 
 import { centrelineKey, ProjectMode, StudyRequestStatus } from '@/lib/Constants';
 import { getStudyRequestBulk } from '@/lib/api/WebApi';
-import { getStudyRequestLocation } from '@/lib/geo/CentrelineUtils';
+import { getStudyRequestInfo, groupRequestsByLocation } from '@/lib/geo/CentrelineUtils';
```

Comment:
groupRequestsByLocation isn't being used anywhere in this file. Do we still need to import it?